sql-statements/sql-statement-split-region.md

```diff
@@ -75,6 +75,13 @@ The following example shows the result of the `SPLIT` statement:
 * `TOTAL_SPLIT_REGION`: the number of newly split Regions.
 * `SCATTER_FINISH_RATIO`: the completion rate of scattering for newly split Regions. `1.0` means that all Regions are scattered. `0.5` means that only half of the Regions are scattered and the rest are being scattered.
 
+> **Note:**
+>
+> The following two session variables might affect the behavior of the `SPLIT` statement:
+>
+> - `tidb_wait_split_region_finish`: It might take a while to scatter the Regions. The duration depends on PD scheduling and TiKV loads. This variable is used to control when executing the `SPLIT REGION` statement whether to return the results to the client until all Regions are scattered. If set to `1` (by default), TiDB returns the value only when the scattering is completed. If set to `0`, TiDB returns the results regardless of the scattering.
```

Comment:
```suggestion
> - `tidb_wait_split_region_finish`: It might take a while to scatter the Regions. This duration depends on PD scheduling and TiKV loads. This variable is used to control when executing the `SPLIT REGION` statement whether to return the results to the client until all Regions are scattered. If its value is set to `1` (by default), TiDB returns the results only after the scattering is completed. If its value is set to `0`, TiDB returns the results regardless of the scattering status.
```